code/game/turfs/closed.dm

```diff
@@ -30,14 +30,21 @@
 /turf/closed/indestructible/splashscreen
 	name = "Space Station 13"
 	icon = 'icons/misc/fullscreen.dmi'
-	icon_state = "title"
+	icon_state = ""
 	layer = FLY_LAYER
 	var/titlescreen = TITLESCREEN
-
-/turf/closed/indestructible/splashscreen/Initialize()
-	..()
-	if(titlescreen)
-		icon_state = pick("title1","title2")
+	var/currenticon
+	var/newicon
+	New() //Do not change this to Initialize().
```

Comment:
very minor nitpick: change this to `/turf/closed/indestructible/splashscreen/New()`